benchmarks/perturbations.py

```diff
@@ -67,5 +69,6 @@ def setup(self):
         self.rho0 = Earth.rho0.to(u.kg / u.km**3).value  # kg/km^3
         self.H0 = Earth.H0.to(u.km).value
         self.tof = 60  # s
-    def time_atmospheric_drag(self):
-        cowell(self.orbit, self.tof, ad=atmospheric_drag, R=self.R, C_D=self.C_D, A=self.A, m=self.m, H0=self.H0, rho0=self.rho0)
+        
+    def time_atmospheric_drag_exponential(self):
+        cowell(self.orbit, self.tof, ad=atmospheric_drag_exponential, R=self.R, C_D=self.C_D, A=self.A, m=self.m, H0=self.H0, rho0=self.rho0)
```

Comment:
Oops, this API changed as well. This should be:

```suggestion
        cowell(self.orbit.attractor.k, self.orbit.r, self.orbit.v, self.tof.reshape(-1), ad=atmospheric_drag_exponential, R=self.R, C_D=self.C_D, A=self.A, m=self.m, H0=self.H0, rho0=self.rho0)
```

and the same for all `cowell` calls.